linebot/response.go

```diff
@@ -93,6 +93,16 @@ type MessageConsumptionResponse struct {
 	TotalUsage int64
 }
 
+// BotInfoResponse type
+type BotInfoResponse struct {
+	UserID         string         `json:"userId"`
+	BasicID        string         `json:"basicId"`
+	PremiumID      string         `json:"premiumId"`
```

Comment:
Do you miss the "displayName"? 

Refer https://developers.line.biz/en/reference/messaging-api/#get-bot-info-response